tsl/test/sql/continuous_aggs_query.sql.in

```diff
@@ -26,19 +26,19 @@ CREATE TABLE conditions (
 
 select table_name from create_hypertable( 'conditions', 'timec');
 
-insert into conditions values ( '2018-01-01 09:00:00-08', 'SFO', 55, 45);
-insert into conditions values ( '2018-01-02 09:00:00-08', 'por', 100, 100);
-insert into conditions values ( '2018-01-02 09:00:00-08', 'SFO', 65, 45);
-insert into conditions values ( '2018-01-02 09:00:00-08', 'NYC', 65, 45);
-insert into conditions values ( '2018-11-01 09:00:00-08', 'NYC', 45, 30);
-insert into conditions values ( '2018-11-01 10:00:00-08', 'NYC', 55, 35);
-insert into conditions values ( '2018-11-01 11:00:00-08', 'NYC', 65, 40);
-insert into conditions values ( '2018-11-01 12:00:00-08', 'NYC', 75, 45);
-insert into conditions values ( '2018-11-01 13:00:00-08', 'NYC', 85, 50);
-insert into conditions values ( '2018-11-02 09:00:00-08', 'NYC', 10, 10);
-insert into conditions values ( '2018-11-02 10:00:00-08', 'NYC', 20, 15);
-insert into conditions values ( '2018-11-02 11:00:00-08', 'NYC', null, null);
-insert into conditions values ( '2018-11-03 09:00:00-08', 'NYC', null, null);
+insert into conditions values ( '2018-01-01 09:20:00-08', 'SFO', 55, 45);
+insert into conditions values ( '2018-01-02 09:30:00-08', 'por', 100, 100);
+insert into conditions values ( '2018-01-02 09:20:00-08', 'SFO', 65, 45);
+insert into conditions values ( '2018-01-02 09:10:00-08', 'NYC', 65, 45);
+insert into conditions values ( '2018-11-01 09:20:00-08', 'NYC', 45, 30);
+insert into conditions values ( '2018-11-01 10:40:00-08', 'NYC', 55, 35);
+insert into conditions values ( '2018-11-01 11:50:00-08', 'NYC', 65, 40);
+insert into conditions values ( '2018-11-01 12:10:00-08', 'NYC', 75, 45);
+insert into conditions values ( '2018-11-01 13:10:00-08', 'NYC', 85, 50);
+insert into conditions values ( '2018-11-02 09:20:00-08', 'NYC', 10, 10);
+insert into conditions values ( '2018-11-02 10:30:00-08', 'NYC', 20, 15);
+insert into conditions values ( '2018-11-02 11:40:00-08', 'NYC', null, null);
+insert into conditions values ( '2018-11-03 09:50:00-08', 'NYC', null, null);
```

Comment:
Nit: Not that it really makes a difference, but why this change? It does not seem warranted.

Reply:
I didn't want to align on hours since bucketing typically falls on some hour of the day. This makes it easier to see that the value returned by the watermark (max-derived value) is a bucket and not an inserted value. I can revert if you feel it is important to keep the same.